### PR TITLE
Update table mobile card design

### LINKED ISSUE #2817: Tidy up spacing on p-table--mobile-card
## Pattern to amend

`p-table--mobile-card
`
## Visual

How it currently looks like:
 
Screenshot 2020-02-06 at 15 21 33

My proposal:
Screenshot 2020-02-06 at 15 23 29

## Context

Visually I don't think it looks right having all that white space on the left side, in my opinion, it looks like a mistake. It's also taking much space compared to my proposal, making the user scroll much more.

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Update table mobile card design to match what was agreed in https://github.com/canonical-web-and-design/vanilla-framework/issues/2817

Fixes #2817

-Drive by: makes muted heading same style as table headers, they are close but slightly different which Ithink is unnecesary.

## QA

- Open docs/examples/patterns/tables/table-mobile-card
- QA

## Screenshots
It is responsive, min column width 12rem, feel free to suggest another value if you think this isn't optimal. Opted for this rather than modifier classes as it is simpler to use.

image
image
image

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

- [x] PR should have one of the following labels to automatically categorise it in release notes:
  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
- [x] Vanilla version in `package.json` should be updated relative to the [most recent release](https://github.com/canonical-web-and-design/vanilla-framework/releases/latest), following semver convention:
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
- [x] Any changes to component class names (new patterns, variants, removed or added features) should be listed on the [component status page](https://github.com/canonical-web-and-design/vanilla-framework/blob/master/templates/docs/component-status.md).
- [x] Documentation side navigation should be updated with the relevant labels.

### CODE REVIEW

File: templates/docs/examples/patterns/tables/table-mobile-card.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {% block standalone_css %}patterns_table-mobile-card{% endblock %}
 
 {% block content %}
-<table class="p-table--mobile-card" aria-label="Example of table transforming into mobile cards on small screens">
+<table class="p-table--mobile-card is-split" aria-label="Example of table transforming into mobile cards on small screens">
```
Comment (bartaz) [inline templates/docs/examples/patterns/tables/table-mobile-card.html:7]: Does `is-split` do anything? I don't see any styling specific for it.

File: scss/_patterns_table-mobile-card.scss [hunk 2]
```diff
@@ -26,46 +27,43 @@
         flex-direction: column;
         margin-bottom: $spv-outer--scaleable;
         overflow: auto; // prevent overflow of child margins
-        padding: 0 $sph-inner--large calc(#{$spv-inner--large} - 1px);
-        width: calc(50% - #{0.5 * map-get($grid-gutter-widths, medium)});
+        padding: 0 $sph-inner;
+        // width: calc(50% - #{0.5 * map-get($grid-gutter-widths, medium)});
```
Comment (bartaz) [inline scss/_patterns_table-mobile-card.scss:31]: Remove if not needed.

Comment (lyubomir-popov) [reply scss/_patterns_table-mobile-card.scss:31]: oh I hadn't saved that file, it is gone now.

File: templates/docs/examples/patterns/tables/table-mobile-card.html (hunk 1 above)
Comment (lyubomir-popov) [reply templates/docs/examples/patterns/tables/table-mobile-card.html]: no, removed

**Comment by bartaz** [comment]
In IE `grid` is not applied, but other styles are.

Screenshot 2021-04-16 at 14 55 03

I think it's fine to ignore IE for mobile card view, so maybe all mobile card related styling should be wrapped in `@supports`, so the table never switches to mobile card view in IE.

**Comment by lyubomir-popov** [comment]
Added ie support @bartaz, ready for another look.

**Review by spencerbygraves — APPROVED** [review]

**Comment by bartaz** [comment]
May not be directly related to this PR, but is more noticable now with new muted heading styling, but when muted heading is used with p-checkbox or p-radio it does not have lighter color, but standard text color:

Screenshot 2021-04-16 at 16 29 04

**Comment by lyubomir-popov** [comment]
done here - https://docs.google.com/document/d/19g_SsOKsPMvkW0Efek2Ys_CadBIb9ddi6lDs7acf9vA/edit

**Comment by lyubomir-popov** [comment]
the headings now look aa they should
image

File: scss/_base_forms-tick-elements.scss [hunk 3]
```diff
@@ -259,6 +259,11 @@ $box-offsets-top: (
 
         display: inline;
         padding-top: 0;
+
+        .p-checkbox__label {
+          // color: $colors--light-theme--text-muted !important;
+          color: pink !important;
```
Comment (bartaz) [inline scss/_base_forms-tick-elements.scss:265]: This doesn't look right ;)

**Review by bartaz — APPROVED** [review]
LGTM, thanks

**Pull request merged**